crates/egui_kittest/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "egui_kittest"
+edition.workspace = true
+license.workspace = true
+rust-version.workspace = true
+version.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+wgpu = ["dep:egui-wgpu", "dep:pollster", "dep:image"]
+snapshot = ["dep:dify"]
+
+default = ["image?/png"]
+
+[dependencies]
+accesskit_consumer = "0.24.0"
+kittest = { git = "https://github.com/rerun-io/kittest", version = "0.1", branch = "main" }
+egui = { workspace = true, features = ["accesskit"] }
+
+# wgpu dependencies
+egui-wgpu = { workspace = true, optional = true }
+pollster = { version = "0.3", optional = true }
+image = { workspace = true, optional = true }
+
+# snapshot dependencies
+dify = { version = "0.7.0", optional = true, default-features = false }
```

Comment:
Add new deps as workspace dependencies under the main `Cargo.toml` (accesskit_consumer, kittest, pollster, dify, …)